autopush/router/webpush.py

```diff
@@ -64,16 +68,18 @@ def preflight_check(self, uaid_data, channel_id):
         """Verifies this routing call can be done successfully"""
         uaid = uaid_data["uaid"]
         if 'current_month' not in uaid_data:
-            self.log.info("Record missing 'current_month' {record}",
-                          record=repr(uaid_data))
+            self.log.info(format="Dropping User", code=102,
```

Comment:
I like that we log the action, should we still log the cause? It's an info so it may help.